presto-hive/src/main/java/com/facebook/presto/hive/HiveS3Config.java

```diff
@@ -151,6 +153,19 @@ public HiveS3Config setS3KmsKeyId(String s3KmsKeyId)
         return this;
     }
 
+    public String getS3SseKmsKeyId()
+    {
+        return s3SseKmsKeyId;
+    }
+
+    @Config("hive.s3.sse.kms-key-id")
```

Comment:
Let's call this `hive.s3.sse-kms.key-id` to since this is the `key-id` for `sse-kms`. This also provides symmetry between the related properties:
* `hive.s3.sse-kms.enabled`
* `hive.s3.sse-kms.key-id`